### PR TITLE
CSOAR-426 || New CSOAR Release Notes section

### PR DESCRIPTION
**Pull request by gbertolinii**
Added new link for the cloud soar release notes section // Added first release note

## Purpose of this pull request

This pull request (PR) ...

Issue number:  CSOAR-426

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by gbertolinii** [comment]
@JainM6 I already did it this morning, but I've done it again just to be safe!

**Comment by JainM6** [comment]
> @JainM6 I already did it this morning, but I've done it again just to be safe!

@gbertolinii Thank you again. I've just added you to the list of contributors, so now you have access.

File: docs/release-notes/index.md [hunk 1]
```diff
@@ -8,5 +8,6 @@ description: Here you'll find a chronological list of the newest Sumo Logic feat
 |:---|:---|
 | [**Service Release Notes**](/release-notes-service) | Latest features and bug fixes for our apps, alerts, security, search, observability, data collectors, and more. [Archive](/release-notes-service/archive), [RSS Feed](https://help.sumologic.com/release-notes-service/rss.xml) |
 | [**Cloud SIEM Release Notes**](/release-notes-cse) | Information about new and enhanced features, updated content (like rules, log mappers and parsers), bug fixes, and other important announcements for Cloud SIEM Enterprise. [Archive](/release-notes-cse/archive), [RSS Feed](https://help.sumologic.com/release-notes-cse/rss.xml) |
+| [**Cloud SOAR Release Notes**](/release-notes-csoar)    | The latest news about CSOAR, like new features, bug fixes, changes to the application and other important announcements for Cloud SOAR. [Archive](/release-notes-csoar/archive) |
```
Comment (jpipkin1) [inline docs/release-notes/index.md:11]:
```suggestion
| [**Cloud SOAR Release Notes**](/release-notes-csoar)    | The latest news about CSOAR, like new features, bug fixes, changes to the application, and other important announcements for Cloud SOAR. [Archive](/release-notes-csoar/archive) |
```

File: blog-csoar/2023-06-08-first-release-note.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Introducing the new Release Notes section for Cloud SOAR
+hide_table_of_contents: true
+image: https://help.sumologic.com/img/sumo-square.png
+---
+
+We welcome you to the new Release Notes section of Cloud SOAR. Here you will find all the latest news about CSOAR, from new features, bug fixes and changes to the application.
```
Comment (jpipkin1) [inline blog-csoar/2023-06-08-first-release-note.md:7]:
```suggestion
We welcome you to the new Release Notes section of Cloud SOAR. Here you will find all the latest news about CSOAR, from new features, bug fixes, and changes to the application.
```

**Review by jpipkin1 — APPROVED** [review]
This looks great, @gbertolinii. Approved.

**Pull request merged**